tests/publiccloud/check_services.pm

```diff
@@ -24,6 +24,12 @@ sub run {
     # Debug
     $instance->ssh_script_run('systemctl --no-pager list-units');
 
+    # Check if there are any failed services
+    my $failed_services = $instance->ssh_script_output('systemctl --failed');
+    unless ($failed_services =~ /0 loaded units listed/) {
```

Comment:
Considering to proceed with this kind of checks here,
given the main scope of this command is to expose the failed services, so that when present, output is somethings like that:
```
> systemctl --failed
  UNIT           LOAD   ACTIVE SUB    DESCRIPTION               
● cockpit.socket loaded failed failed Cockpit Web Service Socket
```
with exit code 0, then I'd kindly suggest to check ` failed ` (with spaces):
```suggestion
    if ($failed_services =~ /\sfailed\s/) {
```
instead of no fail.